culture-portal/src/components/footer/footer.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import "./footer.css";
+import gitIcon from './git.svg';
+
+export default class Footer extends React.Component {
+    state = {
```

Comment:
ESlint shows error to =

Reply:
The error corrected so:
1) npm install eslint babel-eslint --save-dev
2) .eslintrc.js add:
"parser": "babel-eslint",